app/controllers/users_controller.rb

```diff
@@ -25,7 +27,15 @@ def user
     @user ||= User.find(user_params[:id])
   end
 
+  def sort_column
+    UserHelper::SORTABLE_COLUMNS.include?(user_params[:sort]) ? user_params[:sort] : 'first_name'
```

Comment:
As you use it in a controller you can leave this constant in a controller and refer to it in views via the helper method